src/main/java/core/basesyntax/dao/WarehouseDaoWrite.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class WarehouseDaoWrite {
+    public static final String BEGIN_OF_CSV = "fruit,quantity";
+
+    public static void writeData(String fileName, Map<String, Integer> remains) {
```

Comment:
The 'writeData' method should not be static, as it would be better to use dependency injection to provide an instance of the WarehouseDaoWrite class.